src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/http_client_scoped_mock_response_factory_service.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+$container->loadFromExtension('framework', [
```

Comment:
missing several test case IMO. Like what if you have a global one in addition to both a non mocked and a mocked one ?

Reply:
Check now :)